pkg/entity/register/worker.go

```diff
@@ -36,6 +37,7 @@ func NewWorker(
 	reqsToRegisterQueue <-chan fwrequest.EntityFwRequest,
 	reqsRegisteredQueue chan<- fwrequest.EntityFwRequest,
 	maxBatchSize int,
+	maxBatchSizeBytes int,
```

Comment:
not a must have, but proposal idea:
number of fixed value config parameters (max*s) is becoming quite long, we might consider passing a config object instead